src/backends/gr.jl

```diff
@@ -1197,7 +1197,8 @@ function gr_get_legend_geometry(viewport_plotarea, sp)
     x_legend_offset = (viewport_plotarea[2] - viewport_plotarea[1]) / 30
     y_legend_offset = (viewport_plotarea[4] - viewport_plotarea[3]) / 30
 
-    dy = gr_point_mult(sp) * sp[:legendfontsize] * 1.75
+    dy *= get(sp[:extra_kwargs], :legend_hfactor, 1.)
```

Comment:
```suggestion
    dy *= get(sp[:extra_kwargs], :legend_hfactor, 1)
```
If these don't have to be floats for a good reason, integer literals are preferred.